src/include/duckdb.h

```diff
@@ -16,11 +16,15 @@
 //! duplicate of duckdb/main/winapi.hpp
 #ifndef DUCKDB_API
 #ifdef _WIN32
+#ifdef DUCKDB_STATIC
```

Comment:
Maybe this define should be a bit more descriptive?

Reply:
not sure what else though, so maybe its fine

Reply:
I'll push `DUCKDB_STATIC_BUILD`?